report_api/app/report.py

```diff
@@ -27,6 +24,17 @@
     }
 }
 
+app.config.from_pyfile('report.cfg')
```

Comment:
I needed the config line after the new logging definitions so it can use them. Also I thought that imports should logically be all together for clarity